gslib/tests/testcase/unit_testcase.py

```diff
@@ -85,8 +90,9 @@ def setUp(self):
     self.stdout_save = sys.stdout
     self.stderr_save = sys.stderr
     fd, self.stdout_file = tempfile.mkstemp()
-    sys.stdout = os.fdopen(fd, 'w+')
+    sys.stdout = os.fdopen(fd, 'wb+')
     fd, self.stderr_file = tempfile.mkstemp()
+    # do not set sys.stderr to be 'wb+' - it will blow up the logger
```

Comment:
Blow up? Like it will crash, or just start printing garbage, or what?

Reply:
Context: comment by Ed.